data/card_catalogue.json

```diff
@@ -1388,35 +1350,34 @@
         {
           "title": "Holomonk",
           "upgrade": "HoloKicker",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "Summon an immobile monk until end of turn who punches adjacent characters for 6 damage. Deals double damage to characters who are being knocked back. ",
+            "rules": "Summon an immobile monk until end of turn who punches characters who come within range for 6 damage.",
```

Comment:
Simplifying and emphasizing the important part (they hit people who come close, so push people close to them)